<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | Eric Lu Zhang (张璐) </title> <meta name="author" content="Eric Lu Zhang"> <meta name="description" content="publications by categories in reversed chronological order. generated by jekyll-scholar."> <meta name="keywords" content="machine learning, AI for Science, computational biology, bioinformatics, metagenomics"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/lu.png?a019e6b36c96f76dfb5bc746b31d529e"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://ericcombiolab.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> Eric Lu Zhang (张璐) </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">People </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">News </a> </li> <li class="nav-item "> <a class="nav-link" href="/openings/">Openings </a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv </a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description">publications by categories in reversed chronological order. generated by jekyll-scholar.</p> </header> <article> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">Pangaea</abbr> </div> <div id="zhang_exploring_2024" class="col-sm-8"> <div class="title">Exploring high-quality microbial genomes by assembling short-reads with long-range connectivity</div> <div class="author"> Zhenmiao Zhang, Jin Xiao, Hongbo Wang, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Chao Yang, Yufen Huang, Zhen Yue, Yang Chen, Lijuan Han, Kejing Yin, Aiping Lyu, Xiaodong Fang, Lu Zhang' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em>Nature Communications</em>. <em>More Information</em> can be <a href="https://github.com/ericcombiolab/Pangaea" rel="external nofollow noopener" target="_blank">found here</a> , May 2024 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Although long-read sequencing enables the generation of complete genomes for unculturable microbes, its high cost limits the widespread adoption of long-read sequencing in large-scale metagenomic studies. An alternative method is to assemble short-reads with long-range connectivity, which can be a cost-effective way to generate high-quality microbial genomes. Here, we develop Pangaea, a bioinformatic approach designed to enhance metagenome assembly using short-reads with long-range connectivity. Pangaea leverages connectivity derived from physical barcodes of linked-reads or virtual barcodes by aligning short-reads to long-reads. Pangaea utilizes a deep learning-based read binning algorithm to assemble co-barcoded reads exhibiting similar sequence contexts and abundances, thereby improving the assembly of high- and medium-abundance microbial genomes. Pangaea also leverages a multi-thresholding algorithm strategy to refine assembly for low-abundance microbes. We benchmark Pangaea on linked-reads and a combination of short- and long-reads from simulation data, mock communities and human gut metagenomes. Pangaea achieves significantly higher contig continuity as well as more near-complete metagenome-assembled genomes (NCMAGs) than the existing assemblers. Pangaea also generates three complete and circular NCMAGs on the human gut microbiomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="veldsman_structural_2024" class="col-sm-8"> <div class="title">Structural and Functional Disparities within the Human Gut Virome in Terms of Genome Topology and Representative Genome Selection</div> <div class="author"> Werner P. Veldsman, Chao Yang, Zhenmiao Zhang, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Yufen Huang, Debajyoti Chowdhury, Lu Zhang' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em></em> May 2024 </div> <div class="periodical"> Number: 1 Publisher: Multidisciplinary Digital Publishing Institute </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Circularity confers protection to viral genomes where linearity falls short, thereby fulfilling the form follows function aphorism. However, a shift away from morphology-based classification toward the molecular and ecological classification of viruses is currently underway within the field of virology. Recent years have seen drastic changes in the International Committee on Taxonomy of Viruses’ operational definitions of viruses, particularly for the tailed phages that inhabit the human gut. After the abolition of the order Caudovirales, these tailed phages are best defined as members of the class Caudoviricetes. To determine the epistemological value of genome topology in the context of the human gut virome, we designed a set of seven experiments to assay the impact of genome topology and representative viral selection on biological interpretation. Using Oxford Nanopore long reads for viral genome assembly coupled with Illumina short-read polishing, we showed that circular and linear virus genomes differ remarkably in terms of genome quality, GC skew, transfer RNA gene frequency, structural variant frequency, cross-reference functional annotation (COG, KEGG, Pfam, and TIGRfam), state-of-the-art marker-based classification, and phage–host interaction. Furthermore, the disparity profile changes during dereplication. In particular, our phage–host interaction results demonstrated that proportional abundances cannot be meaningfully compared without due regard for genome topology and dereplication threshold, which necessitates the need for standardized reporting. As a best practice guideline, we recommend that comparative studies of the human gut virome always report the ratio of circular to linear viral genomes along with the dereplication threshold so that structural and functional metrics can be placed into context when assessing biologically relevant metagenomic properties such as proportional abundance.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100">LRTK</abbr> </div> <div id="yang_lrtk_2024" class="col-sm-8"> <div class="title">LRTK: a platform agnostic toolkit for linked-read analysis of both human genome and metagenome</div> <div class="author"> Chao Yang, Zhenmiao Zhang, Yufen Huang, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Xuefeng Xie, Herui Liao, Jin Xiao, Werner Pieter Veldsman, Kejing Yin, Xiaodong Fang, Lu Zhang' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em></em> May 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Linked-read sequencing technologies generate high-base quality short reads that contain extrapolative information on long-range DNA connectedness. These advantages of linked-read technologies are well known and have been demonstrated in many human genomic and metagenomic studies. However, existing linked-read analysis pipelines (e.g., Long Ranger) were primarily developed to process sequencing data from the human genome and are not suited for analyzing metagenomic sequencing data. Moreover, linked-read analysis pipelines are typically limited to 1 specific sequencing platform.To address these limitations, we present the Linked-Read ToolKit (LRTK), a unified and versatile toolkit for platform agnostic processing of linked-read sequencing data from both human genome and metagenome. LRTK provides functions to perform linked-read simulation, barcode sequencing error correction, barcode-aware read alignment and metagenome assembly, reconstruction of long DNA fragments, taxonomic classification and quantification, and barcode-assisted genomic variant calling and phasing. LRTK has the ability to process multiple samples automatically and provides users with the option to generate reproducible reports during processing of raw sequencing data and at multiple checkpoints throughout downstream analysis. We applied LRTK on linked reads from simulation, mock community, and real datasets for both human genome and metagenome. We showcased LRTK’s ability to generate comparative performance results from preceding benchmark studies and to report these results in publication-ready HTML document plots.LRTK provides comprehensive and flexible modules along with an easy-to-use Python-based workflow for processing linked-read sequencing datasets, thereby filling the current gap in the field caused by platform-centric genome-specific linked-read data analysis tools.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhai_ruminococcus_2023" class="col-sm-8"> <div class="title">Ruminococcus gnavus plays a pathogenic role in diarrhea-predominant irritable bowel syndrome by increasing serotonin biosynthesis</div> <div class="author"> Lixiang Zhai, Chunhua Huang, Ziwan Ning, and <span class="more-authors" title="click to view 18 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '18 more authors' ? 'Yijing Zhang, Min Zhuang, Wei Yang, Xiaolei Wang, Jingjing Wang, Lu Zhang, Haitao Xiao, Ling Zhao, Pallavi Asthana, Yan Y. Lam, Chi Fung Willis Chow, Jian-dong Huang, Shuofeng Yuan, Kui Ming Chan, Chun-Su Yuan, Johnson Yiu-Nam Lau, Hoi Leong Xavier Wong, Zhao-xiang Bian' : '18 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">18 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Diarrhea-predominant irritable bowel syndrome (IBS-D), a globally prevalent functional gastrointestinal (GI) disorder, is associated with elevated serotonin that increases gut motility. While anecdotal evidence suggests that the gut microbiota contributes to serotonin biosynthesis, mechanistic insights are limited. We determined that the bacterium Ruminococcus gnavus plays a pathogenic role in IBS-D. Monocolonization of germ-free mice with R. gnavus induced IBS-D-like symptoms, including increased GI transit and colonic secretion, by stimulating the production of peripheral serotonin. R. gnavus-mediated catabolism of dietary phenylalanine and tryptophan generated phenethylamine and tryptamine that directly stimulated serotonin biosynthesis in intestinal enterochromafﬁn cells via a mechanism involving activation of trace amine-associated receptor 1 (TAAR1). This R. gnavus-driven increase in serotonin levels elevated GI transit and colonic secretion but was abrogated upon TAAR1 inhibition. Collectively, our study provides molecular and pathogenetic insights into how gut microbial metabolites derived from dietary essential amino acids affect serotonin-dependent control of gut motility.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="wang_benchmarking_2023" class="col-sm-8"> <div class="title">Benchmarking multi-platform sequencing technologies for human genome assembly</div> <div class="author"> Jingjing Wang, Werner Pieter Veldsman, Xiaodong Fang, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Yufen Huang, Xuefeng Xie, Aiping Lyu, Lu Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Genome assembly is a computational technique that involves piecing together deoxyribonucleic acid (DNA) fragments generated by sequencing technologies to create a comprehensive and precise representation of the entire genome. Generating a high-quality human reference genome is a crucial prerequisite for comprehending human biology, and it is also vital for downstream genomic variation analysis. Many efforts have been made over the past few decades to create a complete and gapless reference genome for humans by using a diverse range of advanced sequencing technologies. Several available tools are aimed at enhancing the quality of haploid and diploid human genome assemblies, which include contig assembly, polishing of contig errors, scaffolding and variant phasing. Selecting the appropriate tools and technologies remains a daunting task despite several studies have investigated the pros and cons of different assembly strategies. The goal of this paper was to benchmark various strategies for human genome assembly by combining sequencing technologies and tools on two publicly available samples (NA12878 and NA24385) from Genome in a Bottle. We then compared their performances in terms of continuity, accuracy, completeness, variant calling and phasing. We observed that PacBio HiFi long-reads are the optimal choice for generating an assembly with low base errors. On the other hand, we were able to produce the most continuous contigs with Oxford Nanopore long-reads, but they may require further polishing to improve on quality. We recommend using short-reads rather than long-reads themselves to improve the base accuracy of contigs from Oxford Nanopore long-reads. Hi-C is the best choice for chromosome-level scaffolding because it can capture the longest-range DNA connectedness compared to 10× linked-reads and Bionano optical maps. However, a combination of multiple technologies can be used to further improve the quality and completeness of genome assembly. For diploid assembly, hifiasm is the best tool for human diploid genome assembly using PacBio HiFi and Hi-C data. Looking to the future, we expect that further advancements in human diploid assemblers will leverage the power of PacBio HiFi reads and other technologies with long-range DNA connectedness to enable the generation of high-quality, chromosome-level and haplotype-resolved human genome assemblies.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="xu_accurate_2023" class="col-sm-8"> <div class="title">Accurate and interpretable gene expression imputation on scRNA-seq data using IGSimpute</div> <div class="author"> Ke Xu, ChinWang Cheong, Werner P Veldsman, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Aiping Lyu, William K Cheung, Lu Zhang' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Single-cell ribonucleic acid sequencing (scRNA-seq) enables the quantification of gene expression at the transcriptomic level with single-cell resolution, enhancing our understanding of cellular heterogeneity. However, the excessive missing values present in scRNA-seq data hinder downstream analysis. While numerous imputation methods have been proposed to recover scRNA-seq data, high imputation performance often comes with low or no interpretability. Here, we present IGSimpute, an accurate and interpretable imputation method for recovering missing values in scRNA-seq data with an interpretable instance-wise gene selection layer (GSL). IGSimpute outperforms 12 other state-of-the-art imputation methods on 13 out of 17 datasets from different scRNA-seq technologies with the lowest mean squared error as the chosen benchmark metric. We demonstrate that IGSimpute can give unbiased estimates of the missing values compared to other methods, regardless of whether the average gene expression values are small or large. Clustering results of imputed profiles show that IGSimpute offers statistically significant improvement over other imputation methods. By taking the heart-and-aorta and the limb muscle tissues as examples, we show that IGSimpute can also denoise gene expression profiles by removing outlier entries with unexpectedly high expression values via the instance-wise GSL. We also show that genes selected by the instance-wise GSL could indicate the age of B cells from bladder fat tissue of the Tabula Muris Senis atlas. IGSimpute can impute one million cells using 64 min, and thus applicable to large datasets.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_benchmarking_2023" class="col-sm-8"> <div class="title">Benchmarking genome assembly methods on metagenomic sequencing data</div> <div class="author"> Zhenmiao Zhang, Chao Yang, Werner Pieter Veldsman, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Xiaodong Fang, Lu Zhang' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Metagenome assembly is an efficient approach to reconstruct microbial genomes from metagenomic sequencing data. Although short-read sequencing has been widely used for metagenome assembly, linked- and long-read sequencing have shown their advancements in assembly by providing long-range DNA connectedness. Many metagenome assembly tools were developed to simplify the assembly graphs and resolve the repeats in microbial genomes. However, there remains no comprehensive evaluation of metagenomic sequencing technologies, and there is a lack of practical guidance on selecting the appropriate metagenome assembly tools. This paper presents a comprehensive benchmark of 19 commonly used assembly tools applied to metagenomic sequencing datasets obtained from simulation, mock communities or human gut microbiomes. These datasets were generated using mainstream sequencing platforms, such as Illumina and BGISEQ short-read sequencing, 10x Genomics linked-read sequencing, and PacBio and Oxford Nanopore long-read sequencing. The assembly tools were extensively evaluated against many criteria, which revealed that long-read assemblers generated high contig contiguity but failed to reveal some medium- and high-quality metagenome-assembled genomes (MAGs). Linked-read assemblers obtained the highest number of overall near-complete MAGs from the human gut microbiomes. Hybrid assemblers using both short- and long-read sequencing were promising methods to improve both total assembly length and the number of near-complete MAGs. This paper also discussed the running time and peak memory consumption of these assembly tools and provided practical guidance on selecting them.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="wang_comprehensive_2023" class="col-sm-8"> <div class="title">A comprehensive investigation of statistical and machine learning approaches for predicting complex human diseases on genomic variants</div> <div class="author"> Chonghao Wang, Jing Zhang, Werner Pieter Veldsman, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Xin Zhou, Lu Zhang' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Quantifying an individual’s risk for common diseases is an important goal of precision health. The polygenic risk score (PRS), which aggregates multiple risk alleles of candidate diseases, has emerged as a standard approach for identifying high-risk individuals. Although several studies have been performed to benchmark the PRS calculation tools and assess their potential to guide future clinical applications, some issues remain to be further investigated, such as lacking (i) various simulated data with different genetic effects; (ii) evaluation of machine learning models and (iii) evaluation on multiple ancestries studies. In this study, we systematically validated and compared 13 statistical methods, 5 machine learning models and 2 ensemble models using simulated data with additive and genetic interaction models, 22 common diseases with internal training sets, 4 common diseases with external summary statistics and 3 common diseases for trans-ancestry studies in UK Biobank. The statistical methods were better in simulated data from additive models and machine learning models have edges for data that include genetic interactions. Ensemble models are generally the best choice by integrating various statistical methods. LDpred2 outperformed the other standalone tools, whereas PRS-CS, lassosum and DBSLMM showed comparable performance. We also identified that disease heritability strongly affected the predictive performance of all methods. Both the number and effect sizes of risk SNPs are important; and sample size strongly influences the performance of all methods. For the trans-ancestry studies, we found that the performance of most methods became worse when training and testing sets were from different populations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_privar_2013" class="col-sm-8"> <div class="title">PriVar: a toolkit for prioritizing SNVs and indels from next-generation sequencing data</div> <div class="author"> <em>Lu Zhang</em>, Jing Zhang, Jing Yang, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Dingge Ying, Yu lung Lau, Wanling Yang' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em></em> May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Next-generation sequencing has become a valuable tool for detecting mutations involved in Mendelian diseases. However, it is a challenge to identify the small subset of functionally important mutations from tens of thousands of rare variants in a whole exome/genome. Therefore, we developed a toolkit called PriVar, a systematic prioritization pipeline that takes into consideration calling quality of the variants, their predicted functional impact, known connection of the gene to the disease and the number of mutations in a gene, and inference from linkage analysis.Availability: Executable jar package is available at http://paed.hku.hk/uploadarea/yangwl/html/software.html. Contact: yangwl@hkucc.hku.hkSupplementary information: Supplementary data are available at Bioinformatics online.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="xu_bio_2022" class="col-sm-8"> <div class="title">A machine learning model for disease risk prediction by integrating genetic and non-genetic factors</div> <div class="author"> Yu Xu, Chonghao Wang, Zeming Li, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Yunpeng Cai, Ouzhou Young, Aiping Lyu, Lu Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em></em> May 2022 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="xu_dyndeepdrim_2022" class="col-sm-8"> <div class="title">dynDeepDRIM: a dynamic deep learning model to infer direct regulatory interactions using time-course single-cell gene expression data</div> <div class="author"> Yu Xu, Jiaxing Chen, Aiping Lyu, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'William K Cheung, Lu Zhang' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em></em> May 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Time-course single-cell RNA sequencing (scRNA-seq) data have been widely used to explore dynamic changes in gene expression of transcription factors (TFs) and their target genes. This information is useful to reconstruct cell-type-specific gene regulatory networks (GRNs). However, the existing tools are commonly designed to analyze either time-course bulk gene expression data or static scRNA-seq data via pseudo-time cell ordering. A few methods successfully utilize the information from multiple time points while also considering the characteristics of scRNA-seq data. We proposed dynDeepDRIM, a novel deep learning model to reconstruct GRNs using time-course scRNA-seq data. It represents the joint expression of a gene pair as an image and utilizes the image of the target TF–gene pair and the ones of the potential neighbors to reconstruct GRNs from time-course scRNA-seq data. dynDeepDRIM can effectively remove the transitive TF–gene interactions by considering neighborhood context and model the gene expression dynamics using high-dimensional tensors. We compared dynDeepDRIM with six GRN reconstruction methods on both simulation and four real time-course scRNA-seq data. dynDeepDRIM achieved substantially better performance than the other methods in inferring TF–gene interactions and eliminated the false positives effectively. We also applied dynDeepDRIM to annotate gene functions and found it achieved evidently better performance than the other tools due to considering the neighbor genes.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="chen_deepdrim_2021" class="col-sm-8"> <div class="title">DeepDRIM: a deep neural network to reconstruct cell-type-specific gene regulatory network using single-cell RNA-seq data</div> <div class="author"> Jiaxing Chen, ChinWang Cheong, Liang Lan, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Xin Zhou, Jiming Liu, Aiping Lyu, William K Cheung, Lu Zhang' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em></em> May 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Single-cell RNA sequencing has enabled to capture the gene activities at single-cell resolution, thus allowing reconstruction of cell-type-specific gene regulatory networks (GRNs). The available algorithms for reconstructing GRNs are commonly designed for bulk RNA-seq data, and few of them are applicable to analyze scRNA-seq data by dealing with the dropout events and cellular heterogeneity. In this paper, we represent the joint gene expression distribution of a gene pair as an image and propose a novel supervised deep neural network called DeepDRIM which utilizes the image of the target TF-gene pair and the ones of the potential neighbors to reconstruct GRN from scRNA-seq data. Due to the consideration of TF-gene pair’s neighborhood context, DeepDRIM can effectively eliminate the false positives caused by transitive gene–gene interactions. We compared DeepDRIM with nine GRN reconstruction algorithms designed for either bulk or single-cell RNA-seq data. It achieves evidently better performance for the scRNA-seq data collected from eight cell lines. The simulated data show that DeepDRIM is robust to the dropout rate, the cell number and the size of the training data. We further applied DeepDRIM to the scRNA-seq gene expression of B cells from the bronchoalveolar lavage fluid of the patients with mild and severe coronavirus disease 2019. We focused on the cell-type-specific GRN alteration and observed targets of TFs that were differentially expressed between the two statuses to be enriched in lysosome, apoptosis, response to decreased oxygen level and microtubule, which had been proved to be associated with coronavirus infection.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_metamvgl_2021" class="col-sm-8"> <div class="title">METAMVGL: a multi-view graph-based metagenomic contig binning algorithm by integrating assembly and paired-end graphs</div> <div class="author"> Zhenmiao Zhang, and <em>Lu Zhang</em> </div> <div class="periodical"> <em></em> May 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Due to the complexity of microbial communities, de novo assembly on next generation sequencing data is commonly unable to produce complete microbial genomes. Metagenome assembly binning becomes an essential step that could group the fragmented contigs into clusters to represent microbial genomes based on contigs’ nucleotide compositions and read depths. These features work well on the long contigs, but are not stable for the short ones. Contigs can be linked by sequence overlap (assembly graph) or by the paired-end reads aligned to them (PE graph), where the linked contigs have high chance to be derived from the same clusters.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="yang_review_2021" class="col-sm-8"> <div class="title">A review of computational tools for generating metagenome-assembled genomes from metagenomic sequencing data</div> <div class="author"> Chao Yang, Debajyoti Chowdhury, Zhenmiao Zhang, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'William K. Cheung, Aiping Lu, Zhaoxiang Bian, Lu Zhang' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em></em> May 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Metagenomic sequencing provides a culture-independent avenue to investigate the complex microbial communities by constructing metagenome-assembled genomes (MAGs). A MAG represents a microbial genome by a group of sequences from genome assembly with similar characteristics. It enables us to identify novel species and understand their potential functions in a dynamic ecosystem. Many computational tools have been developed to construct and annotate MAGs from metagenomic sequencing, however, there is a prominent gap to comprehensively introduce their background and practical performance. In this paper, we have thoroughly investigated the computational tools designed for both upstream and downstream analyses, including metagenome assembly, metagenome binning, gene prediction, functional annotation, taxonomic classification, and profiling. We have categorized the commonly used tools into unique groups based on their functional background and introduced the underlying core algorithms and associated information to demonstrate a comparative outlook. Furthermore, we have emphasized the computational requisition and offered guidance to the users to select the most efficient tools. Finally, we have indicated current limitations, potential solutions, and future perspectives for further improving the tools of MAG construction and annotation. We believe that our work provides a consolidated resource for the current stage of MAG studies and shed light on the future development of more effective MAG analysis tools on metagenomic sequencing.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_comprehensive_2020" class="col-sm-8"> <div class="title">A comprehensive investigation of metagenome assembly by linked-read sequencing</div> <div class="author"> <em>Lu Zhang</em>, Xiaodong Fang, Herui Liao, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Zhenmiao Zhang, Xin Zhou, Lijuan Han, Yang Chen, Qinwei Qiu, Shuai Cheng Li' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em></em> May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The human microbiota are complex systems with important roles in our physiological activities and diseases. Sequencing the microbial genomes in the microbiota can help in our interpretation of their activities. The vast majority of the microbes in the microbiota cannot be isolated for individual sequencing. Current metagenomics practices use short-read sequencing to simultaneously sequence a mixture of microbial genomes. However, these results are in ambiguity during genome assembly, leading to unsatisfactory microbial genome completeness and contig continuity. Linked-read sequencing is able to remove some of these ambiguities by attaching the same barcode to the reads from a long DNA fragment (10–100 kb), thus improving metagenome assembly. However, it is not clear how the choices for several parameters in the use of linked-read sequencing affect the assembly quality.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="choobdar_assessment_2019" class="col-sm-8"> <div class="title">Assessment of network module identification across complex diseases</div> <div class="author"> Sarvenaz Choobdar, Mehmet E. Ahsen, Jake Crawford, and <span class="more-authors" title="click to view 20 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '20 more authors' ? 'Mattia Tomasoni, Tao Fang, David Lamparter, Junyuan Lin, Benjamin Hescott, Xiaozhe Hu, Johnathan Mercer, Ted Natoli, Rajiv Narayan, Aravind Subramanian, Jitao D. Zhang, Gustavo Stolovitzky, Zoltán Kutalik, Kasper Lage, Donna K. Slonim, Julio Saez-Rodriguez, Lenore J. Cowen, Roberto Bellotti, Sven Bergmann, Daniel Marbach' : '20 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">20 more authors</span> </div> <div class="periodical"> <em>Nature Methods</em>, May 2019 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Many bioinformatics methods have been proposed for reducing the complexity of large gene or protein networks into relevant subnetworks or modules. Yet, how such methods compare to each other in terms of their ability to identify disease-relevant modules in different types of network remains poorly understood. We launched the ’Disease Module Identification DREAM Challenge’, an open competition to comprehensively assess module identification methods across diverse protein-protein interaction, signaling, gene co-expression, homology and cancer-gene networks. Predicted network modules were tested for association with complex traits and diseases using a unique collection of 180 genome-wide association studies. Our robust assessment of 75 module identification methods reveals top-performing algorithms, which recover complementary trait-associated modules. We find that most of these modules correspond to core disease-relevant pathways, which often comprise therapeutic targets. This community challenge establishes biologically interpretable benchmarks, tools and guidelines for molecular network analysis to study human disease biology.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_assessment_2019" class="col-sm-8"> <div class="title">Assessment of human diploid genome assembly with 10x Linked-Reads data</div> <div class="author"> <em>Lu Zhang</em>, Xin Zhou, Ziming Weng, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Arend Sidow' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1 more author</span> </div> <div class="periodical"> <em></em> May 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Producing cost-effective haplotype-resolved personal genomes remains challenging. 10x Linked-Read sequencing, with its high base quality and long-range information, has been demonstrated to facilitate de novo assembly of human genomes and variant detection. In this study, we investigate in depth how the parameter space of 10x library preparation and sequencing affects assembly quality, on the basis of both simulated and real libraries.We prepared and sequenced eight 10x libraries with a diverse set of parameters from standard cell lines NA12878 and NA24385 and performed whole-genome assembly on the data. We also developed the simulator LRTK-SIM to follow the workflow of 10x data generation and produce realistic simulated Linked-Read data sets. We found that assembly quality could be improved by increasing the total sequencing coverage (C) and keeping physical coverage of DNA fragments (CF) or read coverage per fragment (CR) within broad ranges. The optimal physical coverage was between 332× and 823× and assembly quality worsened if it increased to &gt;1,000× for a given C. Long DNA fragments could significantly extend phase blocks but decreased contig contiguity. The optimal length-weighted fragment length (W\{}mu _{FL}}\) was ∼50–150 kb. When broadly optimal parameters were used for library preparation and sequencing, ∼80% of the genome was assembled in a diploid state.The Linked-Read libraries we generated and the parameter space we identified provide theoretical considerations and practical guidelines for personal genome assemblies based on 10x Linked-Read sequencing.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="guo_coding_2018" class="col-sm-8"> <div class="title">Coding mutations in NUS1 contribute to Parkinson’s disease</div> <div class="author"> Ji-feng Guo, <em>Lu Zhang</em>, Kai Li, and <span class="more-authors" title="click to view 37 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '37 more authors' ? 'Jun-pu Mei, Jin Xue, Jia Chen, Xia Tang, Lu Shen, Hong Jiang, Chao Chen, Hui Guo, Xue-li Wu, Si-long Sun, Qian Xu, Qi-ying Sun, Piu Chan, Hui-fang Shang, Tao Wang, Guo-hua Zhao, Jing-yu Liu, Xue-feng Xie, Yi-qi Jiang, Zhen-hua Liu, Yu-wen Zhao, Zuo-bin Zhu, Jia-da Li, Zheng-mao Hu, Xin-xiang Yan, Xiao-dong Fang, Guang-hui Wang, Feng-yu Zhang, Kun Xia, Chun-yu Liu, Xiong-wei Zhu, Zhen-yu Yue, Shuai Cheng Li, Huai-bin Cai, Zhuo-hua Zhang, Ran-hui Duan, Bei-sha Tang' : '37 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">37 more authors</span> </div> <div class="periodical"> <em></em> May 2018 </div> <div class="periodical"> Publisher: Proceedings of the National Academy of Sciences </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Whole-exome sequencing has been successful in identifying genetic factors contributing to familial or sporadic Parkinson’s disease (PD). However, this approach has not been applied to explore the impact of de novo mutations on PD pathogenesis. Here, we sequenced the exomes of 39 early onset patients, their parents, and 20 unaffected siblings to investigate the effects of de novo mutations on PD. We identified 12 genes with de novo mutations (MAD1L1, NUP98, PPP2CB, PKMYT1, TRIM24, CEP131, CTTNBP2, NUS1, SMPD3, MGRN1, IFI35, and RUSC2), which could be functionally relevant to PD pathogenesis. Further analyses of two independent case-control cohorts (1,852 patients and 1,565 controls in one cohort and 3,237 patients and 2,858 controls in the other) revealed that NUS1 harbors significantly more rare nonsynonymous variants (P = 1.01E-5, odds ratio = 11.3) in PD patients than in controls. Functional studies in Drosophila demonstrated that the loss of NUS1 could reduce the climbing ability, dopamine level, and number of dopaminergic neurons in 30-day-old flies and could induce apoptosis in fly brain. Together, our data suggest that de novo mutations could contribute to early onset PD pathogenesis and identify NUS1 as a candidate gene for PD.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhou_identification_2016" class="col-sm-8"> <div class="title">Identification of RELN variation p.Thr3192Ser in a Chinese family with schizophrenia</div> <div class="author"> Zhifan Zhou, Zhengmao Hu, <em>Lu Zhang</em>, and <span class="more-authors" title="click to view 9 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '9 more authors' ? 'Zhaoting Hu, Haihong Liu, Zhening Liu, Juan Du, Jingping Zhao, Lin Zhou, Kun Xia, Bengsha Tang, Lu Shen' : '9 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">9 more authors</span> </div> <div class="periodical"> <em></em> May 2016 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Schizophrenia (SCZ) is a serious psychiatric disease with strong heritability. Its complexity is reflected by extensive genetic heterogeneity and much of the genetic liability remains unaccounted for. We applied a combined strategy involving detection of copy number variants (CNVs), whole-genome mapping and exome sequencing to identify the genetic basis of autosomal-dominant SCZ in a Chinese family. To rule out pathogenic CNVs, we first performed Illumina single nucleotide polymorphism (SNP) array analysis on samples from two patients and one psychiatrically healthy family member, but no pathogenic CNVs were detected. In order to further narrow down the susceptible region, we conducted genome-wide linkage analysis and mapped the disease locus to chromosome 7q21.13-22.3, with a maximum multipoint logarithm of odds score of 2.144. Whole-exome sequencing was then carried out with samples from three affected individuals and one unaffected individual in the family. A missense variation c.9575 C \textgreater G (p.Thr3192Ser) was identified in RELN, which is known as a risk gene for SCZ, located on chromosome 7q22, in the pedigree. This rare variant, as a highly penetrant risk variant, co-segregated with the phenotype. Our results provide genetic evidence that RELN may be one of pathogenic gene in SCZ.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_genome-wide_2016" class="col-sm-8"> <div class="title">Genome-wide search followed by replication reveals genetic interaction of CD80 and ALOX5AP associated with systemic lupus erythematosus in Asian populations</div> <div class="author"> Yan Zhang, Jing Yang, Jing Zhang, and <span class="more-authors" title="click to view 42 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '42 more authors' ? 'Liangdan Sun, Nattiya Hirankarn, Hai-Feng Pan, Chak Sing Lau, Tak Mao Chan, Tsz Leung Lee, Alexander Moon Ho Leung, Chi Chiu Mok, Lu Zhang, Yongfei Wang, Jiangshan Jane Shen, Sik Nin Wong, Ka Wing Lee, Marco Hok Kung Ho, Pamela Pui Wah Lee, Brian Hon-Yin Chung, Chun Yin Chong, Raymond Woon Sing Wong, Mo Yin Mok, Wilfred Hing Sang Wong, Kwok Lung Tong, Niko Kei Chiu Tse, Xiang-Pei Li, Yingyos Avihingsanon, Pornpimol Rianthavorn, Thavatchai Deekajorndej, Kanya Suphapeetiporn, Vorasuk Shotelersuk, Shirley King Yee Ying, Samuel Ka Shun Fung, Wai Ming Lai, Chun-Ming Wong, Irene Oi Lin Ng, Maria-Merce Garcia-Barcelo, Stacey S. Cherny, Yong Cui, Pak Chung Sham, Sen Yang, Dong-Qing Ye, Xue-Jun Zhang, Yu Lung Lau, Wanling Yang' : '42 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">42 more authors</span> </div> <div class="periodical"> <em></em> May 2016 </div> <div class="periodical"> Publisher: BMJ Publishing Group Ltd Section: Basic and translational research </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2015</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="ai_adaptation_2015" class="col-sm-8"> <div class="title">Adaptation and possible ancient interspecies introgression in pigs identified by whole-genome sequencing</div> <div class="author"> Huashui Ai, Xiaodong Fang, Bin Yang, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Zhiyong Huang, Hao Chen, Likai Mao, Feng Zhang, Lu Zhang, others' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Nature Genetics</em>, May 2015 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Lusheng Huang, Jun Ren and colleagues report the genome sequences of 69 pigs, representing 11 geographically distinct breeds and 3 wild boar populations, from within China. They identify loci related to high- and low-latitude adaptation and infer a likely ancient introgression event in northern Chinese pigs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_association_2015" class="col-sm-8"> <div class="title">Association of Common Variants in LOX with Keratoconus: A Meta-Analysis</div> <div class="author"> Jing Zhang, <em>Lu Zhang</em>, Jiaxu Hong, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'Dan Wu, Jianjiang Xu' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em></em> May 2015 </div> <div class="periodical"> Publisher: Public Library of Science </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Background Several case-control studies have been performed to examine the association of genetic variants in lysyl oxidase (LOX) with keratoconus. However, the results remained inconclusive and great heterogeneity might exist across populations. Method A comprehensive literature search for studies that published up to June 25, 2015 was performed. Summary odds ratios (OR) and 95% confidence intervals (CI) of each single nucleotide polymorphism (SNP) were estimated with fixed effects model when I2\textless50% in the test for heterogeneity or random effects model when I2\textgreater50%. Publication bias was evaluated using funnel plots and Egger’s test. Results A total of four studies including 1,467 keratoconus cases and 4,490 controls were involved in this meta-analysis. SNPs rs2956540 and rs10519694 showed significant association with keratoconus, with ORs of 0.71 (95% CI: 0.63–0.80, P = 1.43E-08) and 0.77 (95% CI: 0.61–0.97, P = 0.026), respectively. In contrast, our study lacked sufficient evidences to support the association of rs1800449/rs2288393 with keratoconus across populations. Conclusion This meta-analysis suggested that two LOX variants, rs2956540 and rs10519694, may affect individual susceptibility to keratoconus, while distinct heterogeneity existed within this locus. Larger-scale and multi-ethnic genetic studies on keratoconus are required to further validate the results.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_meta-analysis_2015" class="col-sm-8"> <div class="title">Meta-analysis of GWAS on two Chinese populations followed by replication identifies novel genetic variants on the X chromosome associated with systemic lupus erythematosus</div> <div class="author"> Yan Zhang, Jing Zhang, Jing Yang, and <span class="more-authors" title="click to view 48 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '48 more authors' ? 'Yongfei Wang, Lu Zhang, Xianbo Zuo, Liangdan Sun, Hai-Feng Pan, Nattiya Hirankarn, Tingyou Wang, Ruoyan Chen, Dingge Ying, Shuai Zeng, Jiangshan Jane Shen, Tsz Leung Lee, Chak Sing Lau, Tak Mao Chan, Alexander Moon Ho Leung, Chi Chiu Mok, Sik Nin Wong, Ka Wing Lee, Marco Hok Kung Ho, Pamela Pui Wah Lee, Brian Hon-Yin Chung, Chun Yin Chong, Raymond Woon Sing Wong, Mo Yin Mok, Wilfred Hing Sang Wong, Kwok Lung Tong, Niko Kei Chiu Tse, Xiang-Pei Li, Yingyos Avihingsanon, Pornpimol Rianthavorn, Thavatchai Deekajorndej, Kanya Suphapeetiporn, Vorasuk Shotelersuk, Shirley King Yee Ying, Samuel Ka Shun Fung, Wai Ming Lai, Chun-Ming Wong, Irene Oi Lin Ng, Maria-Merce Garcia-Barcelo, Stacey S. Cherny, Paul Kwong-Hang Tam, Pak Chung Sham, Sen Yang, Dong Qing Ye, Yong Cui, Xue-Jun Zhang, Yu Lung Lau, Wanling Yang' : '48 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">48 more authors</span> </div> <div class="periodical"> <em></em> May 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Systemic lupus erythematosus (SLE) is a prototypic autoimmune disease that affects mainly females. What role the X chromosome plays in the disease has always been an intriguing question. In this study, we examined the genetic variants on the X chromosome through meta-analysis of two genome-wide association studies (GWAS) on SLE on Chinese Han populations. Prominent association signals from the meta-analysis were replicated in 4 additional Asian cohorts, with a total of 5373 cases and 9166 matched controls. We identified a novel variant in PRPS2 on Xp22.3 as associated with SLE with genome-wide significance (rs7062536, OR = 0.84, P = 1.00E−08). Association of the L1CAM-MECP2 region with SLE was reported previously. In this study, we identified independent contributors in this region in NAA10 (rs2071128, OR = 0.81, P = 2.19E−13) and TMEM187 (rs17422, OR = 0.75, P = 1.47E−15), in addition to replicating the association from IRAK1-MECP2 region (rs1059702, OR = 0.71, P = 2.40E−18) in Asian cohorts. The X-linked susceptibility variants showed higher effect size in males than that in females, similar to results from a genome-wide survey of associated SNPs on the autosomes. These results suggest that susceptibility genes identified on the X chromosome, while contributing to disease predisposition, might not contribute significantly to the female predominance of this prototype autoimmune disease.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="li_outbred_2015" class="col-sm-8"> <div class="title">Outbred genome sequencing and CRISPR/Cas9 gene editing in butterflies</div> <div class="author"> Xueyan Li, Dingding Fan, Wei Zhang, and <span class="more-authors" title="click to view 21 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '21 more authors' ? 'Guichun Liu, Lu Zhang, Li Zhao, Xiaodong Fang, Lei Chen, Yang Dong, Yuan Chen, Yun Ding, Ruoping Zhao, Mingji Feng, Yabing Zhu, Yue Feng, Xuanting Jiang, Deying Zhu, Hui Xiang, Xikan Feng, Shuaicheng Li, Jun Wang, Guojie Zhang, Marcus R. Kronforst, Wen Wang' : '21 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">21 more authors</span> </div> <div class="periodical"> <em></em> May 2015 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>Butterflies are exceptionally diverse but their potential as an experimental system has been limited by the difficulty of deciphering heterozygous genomes and a lack of genetic manipulation technology. Here we use a hybrid assembly approach to construct high-quality reference genomes for Papilio xuthus (contig and scaffold N50: 492kb, 3.4Mb) and Papilio machaon (contig and scaffold N50: 81kb, 1.15Mb), highly heterozygous species that differ in host plant affiliations, and adult and larval colour patterns. Integrating comparative genomics and analyses of gene expression yields multiple insights into butterfly evolution, including potential roles of specific genes in recent diversification. To functionally test gene function, we develop an efficient (up to 92.5%) CRISPR/Cas9 gene editing method that yields obvious phenotypes with three genes, Abdominal-B, ebony and frizzled. Our results provide valuable genomic and technological resources for butterflies and unlock their potential as a genetic model system.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2014</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="qin_whole-genome_2014" class="col-sm-8"> <div class="title">Whole-genome sequencing of cultivated and wild peppers provides insights into Capsicum domestication and specialization</div> <div class="author"> Cheng Qin, Changshui Yu, Yaou Shen, and <span class="more-authors" title="click to view 70 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '70 more authors' ? 'Xiaodong Fang, Lang Chen, Jiumeng Min, Jiaowen Cheng, Shancen Zhao, Meng Xu, Yong Luo, Yulan Yang, Zhiming Wu, Likai Mao, Haiyang Wu, Changying Ling-Hu, Huangkai Zhou, Haijian Lin, Sandra González-Morales, Diana L. Trejo-Saavedra, Hao Tian, Xin Tang, Maojun Zhao, Zhiyong Huang, Anwei Zhou, Xiaoming Yao, Junjie Cui, Wenqi Li, Zhe Chen, Yongqiang Feng, Yongchao Niu, Shimin Bi, Xiuwei Yang, Weipeng Li, Huimin Cai, Xirong Luo, Salvador Montes-Hernández, Marco A. Leyva-González, Zhiqiang Xiong, Xiujing He, Lijun Bai, Shu Tan, Xiangqun Tang, Dan Liu, Jinwen Liu, Shangxing Zhang, Maoshan Chen, Lu Zhang, Li Zhang, Yinchao Zhang, Weiqin Liao, Yan Zhang, Min Wang, Xiaodan Lv, Bo Wen, Hongjun Liu, Hemi Luan, Yonggang Zhang, Shuang Yang, Xiaodian Wang, Jiaohui Xu, Xueqin Li, Shuaicheng Li, Junyi Wang, Alain Palloix, Paul W. Bosland, Yingrui Li, Anders Krogh, Rafael F. Rivera-Bustamante, Luis Herrera-Estrella, Ye Yin, Jiping Yu, Kailin Hu, Zhiming Zhang' : '70 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">70 more authors</span> </div> <div class="periodical"> <em></em> May 2014 </div> <div class="periodical"> Publisher: Proceedings of the National Academy of Sciences </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>As an economic crop, pepper satisfies people’s spicy taste and has medicinal uses worldwide. To gain a better understanding of Capsicum evolution, domestication, and specialization, we present here the genome sequence of the cultivated pepper Zunla-1 (C. annuum L.) and its wild progenitor Chiltepin (C. annuum var. glabriusculum). We estimate that the pepper genome expanded  0.3 Mya (with respect to the genome of other Solanaceae) by a rapid amplification of retrotransposons elements, resulting in a genome comprised of  81% repetitive sequences. Approximately 79% of 3.48-Gb scaffolds containing 34,476 protein-coding genes were anchored to chromosomes by a high-density genetic map. Comparison of cultivated and wild pepper genomes with 20 resequencing accessions revealed molecular footprints of artificial selection, providing us with a list of candidate domestication genes. We also found that dosage compensation effect of tandem duplication genes probably contributed to the pungent diversification in pepper. The Capsicum reference genome provides crucial information for the study of not only the evolution of the pepper genome but also, the Solanaceae family, and it will facilitate the establishment of more effective pepper breeding programs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="fang_genome-wide_2014" class="col-sm-8"> <div class="title">Genome-wide adaptive complexes to underground stresses in blind mole rats Spalax</div> <div class="author"> Xiaodong Fang, Eviatar Nevo, Lijuan Han, and <span class="more-authors" title="click to view 50 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '50 more authors' ? 'Erez Y. Levanon, Jing Zhao, Aaron Avivi, Denis Larkin, Xuanting Jiang, Sergey Feranchuk, Yabing Zhu, Alla Fishman, Yue Feng, Noa Sher, Zhiqiang Xiong, Thomas Hankeln, Zhiyong Huang, Vera Gorbunova, Lu Zhang, Wei Zhao, Derek E. Wildman, Yingqi Xiong, Andrei Gudkov, Qiumei Zheng, Gideon Rechavi, Sanyang Liu, Lily Bazak, Jie Chen, Binyamin A. Knisbacher, Yao Lu, Imad Shams, Krzysztof Gajda, Marta Farré, Jaebum Kim, Harris A. Lewin, Jian Ma, Mark Band, Anne Bicker, Angela Kranz, Tobias Mattheus, Hanno Schmidt, Andrei Seluanov, Jorge Azpurua, Michael R. McGowen, Eshel Ben Jacob, Kexin Li, Shaoliang Peng, Xiaoqian Zhu, Xiangke Liao, Shuaicheng Li, Anders Krogh, Xin Zhou, Leonid Brodsky, Jun Wang' : '50 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">50 more authors</span> </div> <div class="periodical"> <em></em> May 2014 </div> <div class="periodical"> Publisher: Nature Publishing Group </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>The blind mole rat (BMR), Spalax galili, is an excellent model for studying mammalian adaptation to life underground and medical applications. The BMR spends its entire life underground, protecting itself from predators and climatic fluctuations while challenging it with multiple stressors such as darkness, hypoxia, hypercapnia, energetics and high pathonecity. Here we sequence and analyse the BMR genome and transcriptome, highlighting the possible genomic adaptive responses to the underground stressors. Our results show high rates of RNA/DNA editing, reduced chromosome rearrangements, an over-representation of short interspersed elements (SINEs) probably linked to hypoxia tolerance, degeneration of vision and progression of photoperiodic perception, tolerance to hypercapnia and hypoxia and resistance to cancer. The remarkable traits of the BMR, together with its genomic and transcriptomic information, enhance our understanding of adaptation to extreme environments and will enable the utilization of BMR models for biomedical research in the fight against cancer, stroke and cardiovascular diseases.</p> </div> </div> </div> </li> </ol> <h2 class="bibliography">2013</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="zhang_epistatic_2013" class="col-sm-8"> <div class="title">Epistatic Interaction between Genetic Variants in Susceptibility Gene ETS1 Correlates with IL-17 Levels in SLE Patients</div> <div class="author"> Jing Zhang, Yan Zhang, <em>Lu Zhang</em>, and <span class="more-authors" title="click to view 19 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '19 more authors' ? 'Jing Yang, Dingge Ying, Shuai Zeng, Tsz Leung Lee, Chak Sing Lau, Tak Mao Chan, Alexander Moon Ho Leung, Chi Chiu Mok, Sik Nin Wong, Ka Wing Lee, Marco Hok Kung Ho, Pamela Pui Wah Lee, Brian Hon-Yin Chung, Chun Yin Chong, Raymond Woon Sing Wong, Mo Yin Mok, Wilfred Hing Sang Wong, Yu Lung Lau, Wanling Yang' : '19 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">19 more authors</span> </div> <div class="periodical"> <em></em> May 2013 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="abstract hidden"> <p>T-helper cells that produce IL-17 (Th17 cells) are a subset of CD4+ T-cells with pathological roles in autoimmune diseases including systemic lupus erythematosus (SLE), and ETS1 is a negative regulator of Th17 cell differentiation. Our previous work on genome-wide association study (GWAS) identified two variants in the ETS1 gene (rs10893872 and rs1128334) as being associated with SLE. However, like many other risk alleles for complex diseases, little is known on how these genetic variants might affect disease pathogenesis. In this study, we examined serum IL-17 levels from 283 SLE cases and observed a significant correlation between risk variants in ETS1 and serum IL-17 concentration in patients, which suggests a potential mechanistic link between these variants and the disease. Furthermore, we found that the two variants act synergistically in influencing IL-17 production, with evidence of significant genetic interaction between them as well as higher correlation between the haplotype formed by the risk alleles and IL-17 level in patient serum. In addition, the correlation between ETS1 variants and IL-17 level seems to be more significant in SLE patients manifesting renal involvement, dsDNA autoantibody production or early-onset.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="yang_meta-analysis_2013" class="col-sm-8"> <div class="title">Meta-analysis Followed by Replication Identifies Loci in or near CDKN1B, TET3, CD80, DRAM1, and ARID5B as Associated with Systemic Lupus Erythematosus in Asians</div> <div class="author"> </div> <div class="periodical"> <em></em> May 2013 </div> <div class="periodical"> </div> <div class="links"> </div> </div> </div> </li> </ol> <h2 class="bibliography">2010</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> </div> <div id="yang_genome-wide_2010" class="col-sm-8"> <div class="title">Genome-Wide Association Study in Asian Populations Identifies Variants in ETS1 and WDFY4 Associated with Systemic Lupus Erythematosus</div> <div class="author"> Wanling Yang, Nan Shen, Dong-Qing Ye, and <span class="more-authors" title="click to view 29 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '29 more authors' ? 'Qiji Liu, Yan Zhang, Xiao-Xia Qian, Nattiya Hirankarn, Dingge Ying, Hai-Feng Pan, Chi Chiu Mok, Tak Mao Chan, Raymond Woon Sing Wong, Ka Wing Lee, Mo Yin Mok, Sik Nin Wong, Alexander Moon Ho Leung, Xiang-Pei Li, Yingyos Avihingsanon, Chun-Ming Wong, Tsz Leung Lee, Marco Hok Kung Ho, Pamela Pui Wah Lee, Yuk Kwan Chang, Philip H. Li, Ruo-Jie Li, Lu Zhang, Wilfred Hing Sang Wong, Irene Oi Lin Ng, Chak Sing Lau, Pak Chung Sham, Yu Lung Lau, Asian Lupus Genetics Consortium (ALGC)' : '29 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">29 more authors</span> </div> <div class="periodical"> <em></em> May 2010 </div> <div class="periodical"> Publisher: Public Library of Science </div> <div class="links"> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2024 Eric Lu Zhang. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: August 25, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?06cae41083477f121be8cd9797ad8e2f"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.min.js" integrity="sha256-rjmgmaB99riUNcdlrDtcAiwtLIojSxNyUFdl+Qh+rB4=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script src="/assets/js/shortcut-key.js"></script> </body> </html>